lib/simplifyExpression/fractionsSearch/cancelLikeTerms.js

```diff
@@ -291,7 +293,9 @@ function cancelTerms(numerator, denominator) {
   // case 4: the numerator is a constant and denominator is a polynomial term
```

Comment:
Can you update the comment explaining that the denominator has to be a polynomial term with a coefficient, and that we need to ignore cases like 2/a or 2/x^2 (as you explained to me in Slack :D)